utils/speed-regression.tcl

```diff
@@ -20,15 +20,15 @@ proc run-tests branches {
         exec -ignorestderr make 2> /dev/null
 
         if {$branch_id == 0} {
-            puts "  copy redis-benchmark from unstable to /tmp..."
-            exec -ignorestderr cp ./redis-benchmark /tmp
+            puts "  copy placeholderkv-benchmark from unstable to /tmp..."
+            exec -ignorestderr cp ./placeholderkv-benchmark /tmp
```

Comment:
Please use a variable for this and define it in the top of the file.

Example:

```suggestion
            puts "  copy $benchmark from unstable to /tmp..."
            exec -ignorestderr cp ./$benchmark /tmp
```